testnets/noistestnet/chain.json

```diff
@@ -112,13 +112,30 @@
       {
         "address": "https://nois-testnet.rpc.kjnodes.com:443",
         "provider": "kjnodes"
+      },
+      {
+        "address": "https://tnois-rpc.systemd.run:443",
+        "provider": "systemd"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "tnois-grpc.systemd.run:443",
+        "provider": "systemd"
+      },
+      {
+        "address": "nois-testnet-grpc.itrocket.net:21090",
+        "provider": "itrocket"
       }
     ],
-    "grpc": [],
```

Comment:
> better leave this as empty array, to prevent future typo when adding some grpc

Sorry, I don't quite understand how this is supposed to look like.